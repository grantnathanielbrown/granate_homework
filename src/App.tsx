import SearchContainer from "./components/SearchContainer";

export default function App() {

  return (
    <div className="App min-h-screen w-full flex flex-col justify-center items-center text-2xl bg-gradient-to-r from-red-700 to-red-200">
      
      <h1 className="text-6xl m-4">Where are we going to eat?</h1>
      <SearchContainer />
    </div>
  );
}


