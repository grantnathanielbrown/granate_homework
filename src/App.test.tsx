import React from 'react';
import { shallow } from 'enzyme';
import { render, screen } from '@testing-library/react';
import App from './App';

const app = shallow(<App />);
const searchContainer = app.find('SearchContainer');


describe("basic tests", () => {

    it('renders properly', ()  =>  {
        expect(app).toMatchSnapshot();
    });


})

describe()

// expect(screen.getByText("Where are we going to eat?")).toBeInTheDocument();

// test('renders learn react link', () => {
//   render(<App />);
//   const linkElement = screen.getByText(/learn react/i);
//   expect(linkElement).toBeInTheDocument();
// });