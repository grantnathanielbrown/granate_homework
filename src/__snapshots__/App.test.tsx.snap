// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`basic tests renders properly 1`] = `ShallowWrapper {}`;
